sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -59,19 +60,23 @@ import {
   ApiUnprocessableEntityResponse,
 } from "@nestjs/swagger";
 import BaseController from "../BaseController";
+import { IConfig } from "../../types";
 
 @AllowAuthorizedParty(AuthorizedParties.institution)
 @Controller("institution/location")
 @ApiTags("institution")
 export class ProgramInfoRequestController extends BaseController {
+  private readonly config: IConfig;
   constructor(
+    private readonly configService: ConfigService,
```

Comment:
I believe configService doesn't need to be a global variable here. We just pull the config from config service.

![image](https://user-images.githubusercontent.com/54600590/161823547-40601cb3-7d51-4640-ba05-34fe5bdf98e1.png)